src/hdmf/backends/hdf5/h5_utils.py

```diff
@@ -67,6 +67,16 @@ def invert(self):
             self.__inverted = cls(**kwargs)
         return self.__inverted
 
+    def __get_ref(self, ref):
```

Comment:
Doesn't ``__get_ref`` get mangled? This I think will cause trouble with the derived ``AbstractH5TableDataset`` class which uses this function in its ``__init__`` method. I think the method should be renamed to only have a single underscore so that derived classes still have access to it.

Reply:
Good point. I'll fix that.